models/src/main/thrift/content/v1.thrift

```diff
@@ -1683,6 +1684,10 @@ struct ContentChannel {
 
 }
 
+struct SchemaOrg {
+    1: optional list<schemaorg.SchemaRecipe> recipe
+}
+
```

Comment:
Is this better here or in `schemaorg.thrift`? I’m not sure, would appreciate your thoughts.

On the one hand, assuming we add more schemas here in future, if we ever want to reuse this definition in other places it’s probably nicer to have it in `schemaorg.thrift`.

On the other, maybe we don’t want it to be reused anywhere, and it makes more sense to stay here.